tutorials/inverse/55_tfr_stc.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
+"""
+.. _tut-tfr_stc:
+
+====================================
+Time-frequency source reconstruction
+====================================
+
+This tutorial shows how to use source estimation in MNE for time-frequency
+resolved data. And, it shows how to visualize time-frequency source
+estimations using a graphical user interface.
+"""
+# Authors: Alex Rockhill <aprockhill@mailbox.org>
+#
+# License: BSD-3-Clause
+
+# %%
+
+import numpy as np
+import mne
+from mne.datasets import sample
+
+
+# %%
+# Load sensor data and compute time-frequency decomposition
+# ---------------------------------------------------------
+# We will use the sample data set for this tutorial and reconstruct source
+# activity on the trials with a left auditory stimulus.
+#
+# .. note:: You would want to use preprocessing methods such as epoch
+#           rejection and ica artifact rejection (:ref:`tut-reject-data-spans`,
+#           :ref:`tut-artifact-ica` and :ref:`ex-muscle-ica`) to clean the
+#           data for a real analysis. Here we will set extra-strict amplitude
+#           rejection thresholds as a shortcut for only selecting clean trials.
+
+data_path = sample.data_path()
+subjects_dir = data_path / 'subjects'
+raw_fname = data_path / 'MEG' / 'sample' / 'sample_audvis_raw.fif'
+
+# Read the raw data
+raw = mne.io.read_raw_fif(raw_fname)
+raw.info['bads'] = ['MEG 2443']  # bad MEG channel
+
+# Set up the epoching
+event_id = 1  # those are the trials with left-ear auditory stimuli
+tmin, tmax = -0.5, 0.5
+events = mne.find_events(raw)
+
+# pick data channel types
+raw.pick(['meg', 'eeg', 'eog'])
+
+# create epochs, leave room for cutting out edge artifact
+epochs = mne.Epochs(raw, events[events[:, 2] == event_id],
+                    tmin=tmin, tmax=tmax,
+                    baseline=(None, 0), preload=True, proj=False,
+                    reject=dict(grad=4000e-13, mag=4e-12,
+                                eeg=1e-4, eog=150e-6))
+
+# done with eog now
+raw.pick(['meg', 'eeg'])
+epochs.pick(['meg', 'eeg'])
+
+# set average reference
+epochs.set_eeg_reference('average', projection=True)
+
+# compute time-frequency decomposition, focus on alpha and beta oscillations
+freqs = np.logspace(*np.log10([8, 35]), num=6)
+
+# select subset of trials for speed
+epochs = epochs[:10]
+
+# time-frequency decomposition
+epochs_tfr = mne.time_frequency.tfr_morlet(
+    epochs, freqs=freqs, n_cycles=freqs / 2, return_itc=False,
+    average=False, output='complex')
+
+# plot data, there's a lot so you can't see much but we'll see it better
+# later in the tutorial!
+average_tfr = epochs_tfr.average()
+average_tfr.plot_topo(title='Average Amplitude')
+
```

Comment:
I would change this to something like `average_tfr.plot_joint()`, so we do not produce just a white plot. In any case, I would delete the comment - why would you plot something if you "cannot see much"?

Reply:
The comment was from a time recently when the plot worked, it only stopped working a month ago or so.

Reply:
something is definitely wrong with `AverageTFR.plot_topo()`.  I'll open an issue about that.